_temp/hud/fake/fake-request-service.js

```diff
@@ -4,11 +4,10 @@ var $ = require('$jquery');
 var chance = require('../../../fake/fake-extension'); 
 
 $.getJSON = function(url, data, callback) {
-	// TODO: turn into a strategy latere on if needed
-	if (url == '/glimpse/context/?contextId=1234&types=environment,user-identification,web-response,web-request,after-action-invoked,after-action-view-invoked,before-execute-command,after-execute-command,after-view-component') {
-		setTimeout(function() {
-			callback(chance.mvcRequest(new Date()).messages);
-		}, 1000);
-		
-	}
+    // TODO: turn into a strategy latere on if needed
+    if (url == '/glimpse/context/?contextId=1234&types=environment,user-identification,web-response,web-request,after-action-invoked,after-action-view-invoked,before-execute-command,after-execute-command,after-view-component') {
```

Comment:
Actually, not going to change. Updates to the fake data generator are moe than what I want to cover here.